src/ResourceManager/Compute/Commands.Compute/VirtualMachine/VirtualMachineBaseCmdlet.cs

```diff
@@ -186,6 +186,7 @@ private static int GetTabLength(Object obj, int max, int depth, List<Tuple<strin
                                 else
                                 {
                                     tupleList.Add(MakeTuple(property.Name, GetChildProperties(childObject), depth));
+                                    max = Math.Max(max, property.Name.Length);
```

Comment:
@cormacpayne  it is used in format file.  https://github.com/Azure/azure-powershell/blob/dev/src/ResourceManager/Compute/Commands.Compute/Microsoft.Azure.Commands.Compute.format.generated.ps1xml